MAINTENANCE.md

```diff
@@ -0,0 +1,75 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
+- Having the final say in decisions for the tagging schema.
+- Creating releases.
+- Updating dependencies.
+
+and all the following roles.
+
+### Contributor Role
+
+- [Kyle](https://github.com/k-yle) contributes to this project as a volunteer.
+- [Tobias](https://github.com/tordans) contributes to this project as a volunteer.
+
+The contributor role includes:
+- Reviewing PRs.
+- Merging "clear-cut" PRs.
+
+and all the following roles.
+
+### Triage Role
+
+(No one has the triage role at the moment. Please reach out if you want to help.)
+
+The triage role includes:
+- Helping to clarify issues and PRs.
+- Closing issues as duplicates or not planned.
+
+### Contributors
+
+To all contributors, thank you so much for your support! ❤ Especially for:
+- Suggesting new presets and fields or updates to the repository.
+- Researching and helping with issues and PRs.
+- Translating the tagging schema.
+
+Code contributions: [Check this complete list of contributors on GitHub](https://github.com/openstreetmap/id-tagging-schema/graphs/contributors).
+
+## Processes
+
+### PR Reviews and Merges
+
+- PRs need approval from two people: the author and one more contributors before being merged.
```

Comment:
```suggestion
- PRs need approval from two people: the author and one or more contributors before being merged.
```

Reply:
There are other ways to amend this. BTW, is "contributor" requiring any experience/presence? Past accepted PR? It is kind of unclear

Reply:
> BTW, is "contributor" requiring any experience/presence? Past accepted PR? It is kind of unclear

I just added a very high level section to handle the question how the defined roles are assigned https://github.com/openstreetmap/id-tagging-schema/pull/1230/files#diff-87a0c4623bffa64c60b62a39eeab1c40bd0f329cebc177f5e4492555fb5d42aeR68-R73
This does not go into details on which criteria are used to make this decision which is IMO fine because in the absence of published criteria it is clear IMO that the person responsible for the step will have to make a judgement decision. 

Pink @tyrasd